packages/next/build/webpack-config.ts

```diff
@@ -824,13 +824,14 @@ export default async function getBaseWebpackConfig(
       nodeEnv: false,
       splitChunks: isServer
         ? isWebpack5
-          ? {
+          ? ({
+              filename: `${dev ? '[name]' : '[name].[contenthash]'}.js`,
```

Comment:
```suggestion
              filename: '[name].js',
```

Server doesn't use contenthash?

Reply:
We use it for `chunkFilename` here https://github.com/vercel/next.js/blob/7426ebcbc3802cd05b99cd964c56088d87ad5589/packages/next/build/webpack-config.ts#L933

Reply:
Ok